oidc/src/main/java/no/nav/common/oidc/auth/OidcAuthenticatorConfig.java

```diff
@@ -16,13 +18,18 @@
     // Client ID / Audience
     public String clientId;
 
-    // Name of the cookie where the id token should be stored
-    public String idTokenCookieName;
-
     // What type of user is being authenticated
     public IdentType identType;
 
-    // Name of the cookie where the refresh token should be stored (optional)
+    // Name of the cookie where the users id token is stored (optional)
+    // If provided: will try to retrieve id token from cookie before using "idTokenFinder"
+    // If provided: will be used to set refreshed id tokens
+    public String idTokenCookieName;
+
+    // Retrieves the id token from incoming requests (optional)
```

Comment:
Forslag til doc: Override retrieval of the id token from incoming requests. Defaults to Bearer token in Authorization header. (optional)

Reply:
Teknisk sett så overrider man ikke uthentingen helt, kun fallbacket etter man potensielt har prøvd å hente token fra cookie